grails-app/controllers/au/org/ala/collectory/GbifController.groovy

```diff
@@ -184,6 +181,7 @@ class GbifController {
     )
     @Path("/ws/gbif/scan/{uid}")
     @Produces("application/json")
+    @AlaSecured(["ROLE_EDITOR"])
```

Comment:
Use `gbifRegistrationRole` instead of ROLE_EDITOR. Also permit m2m authorised scope.